apps/backoffice-v2/src/common/components/organisms/Header/Header.Navbar.tsx

```diff
@@ -53,10 +53,10 @@ export const Navbar: FunctionComponent = () => {
       <ul className={`menu menu-compact w-full space-y-2`}>
         {filters?.map(({ id, name, entity }) => (
           <NavItem
-            href={`/en/case-management/entities?entity=${entity}&filterId=${id}&filterName=${name}`}
+            href={`/en/case-management/entities?entity=${entity}&filterId=${id}`}
             key={id}
             className={ctw(`capitalize`, {
-              'bg-muted font-bold': name === searchParams?.filterName,
+              'bg-muted font-bold': id === searchParams?.filterId,
```

Comment:
Unrelated fix:
We added the `filterName` to the query params just for comparing the current filter. It was unnecessary because we already have `filterId`.